<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainOperate</class>
 <widget class="QMainWindow" name="MainOperate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainOperate</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="start_wall">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="blur_group">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>231</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>窗口模糊方式</string>
     </property>
     <widget class="QRadioButton" name="radio_blur1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>模糊1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_blur2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>模糊2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_trans">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>透明</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
